cpp/src/parquet/arrow/schema.cc

```diff
@@ -424,8 +425,13 @@ Status NodeToFieldInternal(const Node& node,
   return Status::OK();
 }
 
+std::shared_ptr<Field> ToDictionary32(const Field& field) {
```

Comment:
I don't see where this is used?

Reply:
I'm planning to do some refactoring related to the schema conversion and instantiation of column readers, will return to this then